src/Apps/Collect/Routes/Collection/Components/Header/index.tsx

```diff
@@ -22,14 +12,23 @@ import {
   Serif,
   Spacer,
 } from "@artsy/palette"
-
-import { AuthIntent, ContextModule } from "@artsy/cohesion"
 import { Header_artworks } from "__generated__/Header_artworks.graphql"
 import { Header_collection } from "__generated__/Header_collection.graphql"
+import { CollectionDefaultHeaderFragmentContainer as CollectionDefaultHeader } from "Apps/Collect/Routes/Collection/Components/Header/DefaultHeader"
+import { HorizontalPadding } from "Apps/Components/HorizontalPadding"
 import { useSystemContext } from "Artsy"
+import { AnalyticsSchema } from "Artsy/Analytics"
+import { unica } from "Assets/Fonts"
```

Comment:
If this is being used somewhere in the file it seems problematic. All type should flow through palette's typography components directly.

Reply:
Will do (these weren't actually added, just resorted)